src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -100,8 +100,10 @@
 long? JammedLastAge => Jammed ? 0 : (Host.GetTimeContinuousMilli() - JammedLastTime) / 1000;
 
 int?	ShieldHpPercent => ShipUi?.HitpointsAndEnergy?.Shield / 10;
+int droneFightingCounter = 0;
```

Comment:
Why is `droneFightingCounter ` scoped to the whole script?
What is a case where its state is actually reused between method calls?